packages/app-store/AppDependencyComponent.tsx

```diff
@@ -13,7 +12,7 @@ export const AppDependencyComponent = ({
   dependencyData,
 }: {
   appName: string;
-  dependencyData: RouterOutputs["viewer"]["apps"]["queryForDependencies"];
+  dependencyData?: { name: string; slug: string; installed: boolean }[];
```

Comment:
makes sense for this to be optional since the business logic below checks whether it's truthy or not